### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by ninsaccplayzyt**
Added the following to the commands part of docs:
- afk
-  gafk
- ballot
- createreactionrole

### CODE REVIEW

File: docs/app/(docs)/commands/ballot/page.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Command: `ballot` - SudoBot"
+short_name: ballot
+---
+
+import Modes from "@/components/MDX/Modes";
+
+# `ballot`
+
+Turns anti join on or off. Makes it so that if someone joins while it's activated, they get kicked from the server.
```
Comment (virtual-designer) [inline docs/app/(docs)/commands/ballot/page.mdx:10]: Pretty sure this is a mistake lol. Would you mind correcting this, or I can make a patch if you want!

**Review by virtual-designer — CHANGES_REQUESTED** [review]
```diff
- Turns anti join on or off. Makes it so that if someone joins while it's activated, they get kicked from the server.
+ Allows you to manage ballots. Ballots are polls, where you can cast a vote about something.
```

File: docs/app/(docs)/commands/ballot/page.mdx (hunk 1 above)
Comment (ninsaccplayzyt) [reply docs/app/(docs)/commands/ballot/page.mdx:10]: That was a mistake lol, I'll correct it now, one minute.

Comment (ninsaccplayzyt) [reply docs/app/(docs)/commands/ballot/page.mdx]: Fixed!

**Review by virtual-designer — APPROVED** [review]
LGTM! Thanks a lot for your hard work!

**Pull request merged**